### PR TITLE
Add status querying ETH transactions in frontend

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

image

image

image

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by lukicenturi** [comment]
I temporarily pushed @yabirgb commits too, since I changed something on his commit (to remove conflict while checking). Then I will drop his commits once his commits is merged to develop.

**Review by LefterisJP — COMMENTED** [review]
Frontend UI appearance looks good to me.

@kelsos please review the code.

**Review by kelsos — APPROVED** [review]
looks good thanks

File: frontend/app/package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.16.7",
     "@babel/plugin-proposal-optional-chaining": "7.16.7",
     "@babel/preset-env": "7.16.11",
-    "@intlify/eslint-plugin-vue-i18n": "1.2.0",
+    "@intlify/eslint-plugin-vue-i18n": "2.0.0",
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/package.json:62]: Yes the previous version wasn't supporting ignores

File: frontend/app/src/views/history/transactions/TransactionQueryStatus.vue [hunk 2]
```diff
@@ -0,0 +1,427 @@
+<template>
+  <tr v-if="length > 0" :class="$style.tr">
+    <td :colspan="$vuetify.breakpoint.xsOnly ? 2 : colspan" class="py-2">
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/src/views/history/transactions/TransactionQueryStatus.vue:3]: LOL many other pages write it this way. 
I think we can change all of them later? @kelsos

[the remark this answers is not in the document]
Comment (kelsos) [reply frontend/app/src/views/history/transactions/TransactionQueryStatus.vue:3]: Yes that was what we did in the beginning, well no need to change right now we can re-evaluate along with moving to vuetify 3 probably

**Pull request merged**